packages/zowe-explorer/README.md

```diff
@@ -125,7 +115,7 @@ Use the Log in to the **Authentication Service** feature to regenerate a new tok
 
    You are prompted to enter your username and password.
 
-The token is stored in the default base profile .yaml file.
+The token is stored in the default base profile file, YAML file for v1 Profiles or the Team configuration file.
```

Comment:
Since we allow multiple base profiles depending on which layer the user may be interacting with (project, vs global), should we say `corresponding base profile` 

Are we using the term "_Team configuration file_" interchangeably with "_configuration file for **Team Profiles**_" ?
I have no strong preference for either one (since I personally use both terms) 😋